OpenMEEG/libs/OpenMEEG/include/integrator.h

```diff
@@ -177,7 +177,7 @@ namespace OpenMEEG {
 
     static const unsigned nbPts[4] = {3, 6, 7, 16};
 
-    template <class T, class I>
```

Comment:
I do not understand why the renaming is useful. Not a reason to block the merge though. Note also that "class" should be replace with typename one day or the other.